src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -100,6 +100,66 @@
+           (style/create-form-label "Method Configuration Namespace")
+           (style/create-text-field {:style {:width "100%"} :ref "mcNamespace"
+                                     :defaultValue (get-in (:config props) ["namespace"])})
+           (style/create-form-label "Method Configuration Name")
+           (style/create-text-field {:style {:width "100%"} :ref "mcName"
+                                     :defaultValue (get-in (:config props) ["name"])})
+           [:div {:style {:marginTop 40 :textAlign "center"}}
+            [:a {:style {:marginRight 27 :marginTop 2 :padding "0.5em"
+                         :display "inline-block"
+                         :fontSize "106%" :fontWeight 500 :textDecoration "none"
+                         :color (:button-blue style/colors)}
+                 :href "javascript:;"
+                 :onClick #(clear-overlay state refs)
+                 :onKeyDown (common/create-key-handler [:space :enter] #(clear-overlay state refs))}
+             "Cancel"]
+            (when (:creating-wf @state)
+                  [comps/Blocker {:banner "Publishing Method Configuration..."}])
+            [comps/Button {:text "Publish" :ref "publishButton"
+                           :onClick
+                                 #(let [ns (-> (@refs "mcNamespace") .getDOMNode .-value clojure.string/trim)
+                                        n (-> (@refs "mcName") .getDOMNode .-value clojure.string/trim)]
+                                       (when-not (or (empty? ns) (empty? n))
+                                                 (swap! state assoc :creating-wf true)
+                                                 (endpoints/call-ajax-orch
+                                                   {:endpoint (endpoints/copy-method-config-to-repo (:workspace-id props) (:config props))
+                                                    :headers {"Content-Type" "application/json"}
+                                                    :payload  {:configurationNamespace ns,
+                                                               :configurationName n,
+                                                               :sourceNamespace (get-in (:config props) ["namespace"]),
+                                                               :sourceName (get-in (:config props) ["name"])}
+                                                    :on-done  (fn [{:keys [success?]}]
+                                                                  (swap! state dissoc :creating-wf)
+                                                                  (if success?
+                                                                    (do (clear-overlay state refs))
+                                                                    (js/alert "Method Configuration Publish Failed.")))})))}]]]]))
+
+(react/defc PublishButton
+  {:render (fn [{:keys [props state refs]}]
+             [:div ()
```

Comment:
Just noticed this... Change to `[:div {}`.  What you have works... but only because of some apparent leniency in React.